qtconsole/console_widget.py

```diff
@@ -396,6 +399,7 @@ def eventFilter(self, obj, event):
             text widgets.
         """
         etype = event.type()
+        self._trigger_is_complete_callback()
```

Comment:
Great, thanks. That answers my concern, then. I'll let @ccordoba12 or @wmvanvliet review it in more detail.